contributor-guide/modules/ROOT/pages/release-notes.adoc

```diff
@@ -1,5 +1,129 @@
+
+=== New Features
+
+Describe any new features or major enhancements. For complex features, consider adding a brief example or pointing to updated documentation. If the feature is referenced in an *Issue*, then consider adding a link to that issue.
+
+=== Bug Fixes
+
+List the fixed bugs, ideally with a brief description. If you're using a bug tracking system (like GitHub issues), provide links to the bug reports described in the issues. Include the identifier or number of the bug if available.
+
+=== Deprecations
+
+Mention any functions, classes, or features that are deprecated and might be removed in future releases.
+
+=== Performance Improvements
+
+Detail any optimizations or performance-related changes, ideally with metrics or benchmarks if they are significant.
+
+=== Documentation Updates
+
+Highlight and link to any significant updates in documentation, new examples, or tutorials.
+
+=== Acknowledgements
+
+Credit contributors or those who reported crucial bugs. Recognize any person or organization that played a special role in this release.
+
+=== Known Issues
+
+Briefly describe any known limitations or issues that are still unresolved in the current release. Include known compatibility issues - including with compilers and tools, and other libraries.
+
+=== Upgrade Notes
+
+Provide any specific instructions or recommendations for users upgrading from a previous version, especially if there are any manual steps involved.
+
+== Consistency and Formatting
+
+* Use clear and concise language. Experience has shown that short release notes are read, long files much less so. Do not add into release notes extensive text that should be in your main library documentation.
+
+* Be consistent in the formatting. If you're using bullet points for one section, maintain that for others.
+* Use headers and sub-headers to clearly differentiate sections.
+* Provide links to:
+[circle]
+    ** Your main library documentation, if there is updated text, examples or tutorials
+    ** Relevant issues, discussions or threads
+
+== Examples
+
+The following examples show some different approaches. Note how many rely on links to issues to provide the detailed information.
+
+* https://www.boost.org/doc/libs/1_83_0/libs/json/doc/html/json/release_notes.html[JSON Release Notes]
+* https://www.boost.org/doc/libs/1_83_0/libs/nowide/doc/html/changelog_page.html[NoWide Change Log]
+* https://www.boost.org/doc/libs/1_83_0/libs/type_traits/doc/html/boost_typetraits/history.html[Type Traits History]
+* https://www.boost.org/doc/libs/1_81_0/libs/variant2/doc/html/variant2.html#changelog[Variant2 Revision History]
+
+== Update Boost History
+
+When you have completed the library release notes, add the required information to the `[section New Libraries]` or `[section Updated Libraries]` of the https://github.com/boostorg/website/tree/master/feed/history[boostorg/website/feed/history]. Copy the formatting of the examples below.
```

Comment:
It's worth mentioning this is quickbook format (https://www.boost.org/doc/libs/1_83_0/doc/html/quickbook.html) because new contributors might not know about it at this point.

Reply:
Reference added.